test/e2eutil/wait_util.go

```diff
@@ -314,7 +315,8 @@ func ExecPod(f *framework.Framework, namespace string, pod *corev1.Pod, cmd []st
 
 }
 
-func HelperInitCassandraConfigMap(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, configMapName, namespace string) {
+func HelperInitCassandraConfigMap(t *testing.T, f *framework.Framework, ctx *framework.Context,
+	configMapName, namespace string) {
```

Comment:
```suggestion
func HelperInitCassandraConfigMap(t *testing.T, f *framework.Framework, ctx *framework.Context, configMapName, namespace string) {
```
Another one liner

Reply:
I split all those lines because they exceed the recommended line width